e2e_test.go

```diff
@@ -17,7 +18,11 @@ var fuzz bool
 func init() {
 	flag.BoolVar(&e2e, "e2e", false, "run the end tests (requires a working influxdb instance on 127.0.0.1)")
 	flag.BoolVar(&fuzz, "fuzz", false, "(Not Implemented) run the end tests (requires a working influxdb instance on 127.0.0.1)")
+}
+
+func TestMain(m *testing.M) {
 	flag.Parse()
+	os.Exit(m.Run())
```

Comment:
Take it or leave it: `testing` package calls `flag.Parse()` automatically. So entire test main here can be dropped.